app/views/incomes/YourIncomeCalculationView.scala.html

```diff
@@ -32,101 +32,104 @@
+                                <h2 class="govuk-heading-m">@Messages("tai.income.calculation.heading",s"${Dates.formatDate(TaxYear().start)}",s"${Dates.formatDate(TaxYear().end)}")</h2>
                             }
 
-                            <p>@Messages("tai.income.calculation.rti.ceased.emp",s"${DateHelper.toDisplayFormat(model.endDate)}")</p>
+                            <p class="govuk-body">@Messages("tai.income.calculation.rti.ceased.emp",s"${DateHelper.toDisplayFormat(model.endDate)}")</p>
                         }
 
                         case _ => {
                             @if(model.latestPayment.isDefined) {
-                                <h2 class="heading-medium">@Messages("tai.income.calculation.heading.withRti", Dates.formatDate(model.latestPayment.get.date))</h2>
+                                <h2 class="govuk-heading-m">@Messages("tai.income.calculation.heading.withRti", Dates.formatDate(model.latestPayment.get.date))</h2>
                             } else {
-                                <h2 class="heading-medium">@Messages("tai.income.calculation.heading",s"${Dates.formatDate(TaxYear().start)}",s"${Dates.formatDate(TaxYear().end)}")</h2>
+                                <h2 class="govuk-heading-m">@Messages("tai.income.calculation.heading",s"${Dates.formatDate(TaxYear().start)}",s"${Dates.formatDate(TaxYear().end)}")</h2>
                             }
                         }
                     }
                 </div>
 
                 @if(model.latestPayment.isDefined) {
                     <div class="section soft--top soft--bottom">
-                        <table id="taxable-income-table" class="table-section">
-                            <caption class="text--left table__caption table__caption--bottom-border visually-hidden">@Messages("tai.income.calculation.incomeTable.caption")</caption>
-                            <thead>
-                                <tr>
-                                    <th>@Messages("tai.income.calculation.incomeTable.dateHeader")</th>
-                                    <th class="text--right">@Messages("tai.income.calculation.incomeTable.incomeHeader")</th>
-                                    <th class="text--right">@Messages("tai.income.calculation.incomeTable.taxPaidHeader")</th>
-                                    <th class="text--right">@Messages("tai.income.calculation.incomeTable.nationalInsuranceHeader")</th>
+                        <table id="taxable-income-table" class="govuk-table">
+                            <caption class="text--left table__caption table__caption--bottom-border govuk-visually-hidden">@Messages("tai.income.calculation.incomeTable.caption")</caption>
+                            <thead class="govuk-table__head">
+                                <tr class="govuk-table__row">
+                                    <th scope="col" class="govuk-table__header">@Messages("tai.income.calculation.incomeTable.dateHeader")</th>
```

Comment:
are the scopes really needed?

Reply:
Going off of the gds pattern they are. Up for debate; however, as to whether they're required - passes HTML Validator check :)

Reply:
yep needed  for screen readers